### PR TITLE
[DOCS] Add Dev Tools warning to Security App API subpages

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses  #646. 

Preview [here](https://security-docs_686.docs-preview.app.elstc.co/guide/en/security/master/security-apis.html).

### CODE REVIEW

**Review by jmikell821 — APPROVED** [review]

File: docs/detections/api/rules/rules-api-delete.asciidoc [hunk 1]
```diff
@@ -2,6 +2,7 @@
 === Delete rule
 
 Deletes a single rule using the `rule_id` or `id` field.
+NOTE: The {kib} Console supports only Elasticsearch APIs. You cannot interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```
Comment (peluja1012) [inline docs/detections/api/rules/rules-api-delete.asciidoc:5]: This looks a bit off. May all we need is a new line to fix it?
image

File: docs/detections/api/rules/index-api-overview.asciidoc [hunk 2]
```diff
@@ -3,14 +3,15 @@
 
 You use the index endpoint to create, get, and delete
 `.siem-signals-<Kibana-space>` system indices in a {kib} space.
+NOTE: The {kib} Console supports only Elasticsearch APIs. You cannot interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```
Comment (peluja1012) [inline docs/detections/api/rules/index-api-overview.asciidoc:6]: This one also looks a bit off:
image

File: docs/events/api/timeline-api-import.asciidoc [hunk 3]
```diff
@@ -2,6 +2,7 @@
 === Import timelines and timeline templates
 
 Imports timelines and timeline templates from an `ndjson` file.
+NOTE: The {kib} Console supports only Elasticsearch APIs. You cannot interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```
Comment (peluja1012) [inline docs/events/api/timeline-api-import.asciidoc:5]: This one looks a bit off too:
image

**Review by peluja1012 — APPROVED** [review]
Thanks for these updates @nastasha-solomon! It looks good to me after we fix the couple of small spacing issues.

**Pull request merged**